x-pack/metricbeat/module/aws/utils.go

```diff
@@ -18,6 +18,9 @@ import (
 	resourcegroupstaggingapitypes "github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi/types"
 )
 
+const DefaultApiTimeout = 5 * time.Second
+const CwApiTimeout = 10 * time.Second
```

Comment:
i'm wondering if we should _not_ add a timeout for CW. i really don't have a sense of how long some of the larger metric queries can take. they're paged at least right, so it shouldn't be super long, but still i'm not sure we have a good idea of what a sensible value is here. what do you think ?

Reply:
It would have made sense to have a timeout, even for CW calls, to be consistent. However, setting a hard timeout might be premature as these calls are blocking (i.e., we do not publish events in case of failures), and we need to understand how long they can take.